testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -4334,11 +4390,36 @@ fn partial_tenure_fork() {
         });
         btc_blocks_mined += 1;
 
+        let tip_sn = SortitionDB::get_canonical_burn_chain_tip(sortdb.conn()).unwrap();
+        let sortition_winner = match tip_sn.miner_pk_hash {
+            Some(pk_hash) => {
+                if pk_hash == mining_pkh_1 {
+                    1
+                } else {
+                    2
+                }
+            }
+            // No sortition means miner 2 won (?)
```

Comment:
I think no sortition would just mean there was a problem, not that miner 2 won. There should always be a sortition.

Reply:
Ah yeah pretty much all of your comments here can be addressed with "yeah, this is me trying to understand stuff and not getting it to work" lol